doc/command_injection.rdoc

```diff
@@ -0,0 +1,29 @@
+== Command Injection
+
+Some Ruby core methods accept string data
+that includes text to be executed as a system command.
+
+They should not be called with unknown or unsanitized commands.
+
+These methods include:
+
+- Kernel.system
+- {`command` (backtick method)}[../Kernel.html#method-i-60]
```

Comment:
This prefix should be removed since the `doc/` page directory will be stripped.
```suggestion
- {`command` (backtick method)}[Kernel.html#method-i-60]
```

Reply:
Since 88b1d21dbb64b4a310e589b1ac8b7661ea09500f, it simply can be:
```suggestion
- {`command` (backtick method)}[rdoc-ref:Kernel#`]
```